<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CStyleWidget</class>
 <widget class="QWidget" name="CStyleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>皮肤管理器</string>
  </property>
  <property name="windowOpacity">
   <double>2.000000000000000</double>
  </property>
  <widget class="QTextEdit" name="txt_Content">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>441</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Name">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>440</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Add">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Delete">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Modify">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Apply">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>应用</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>441</width>
     <height>52</height>
    </rect>
   </property>
   <property name="title">
    <string>风格</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>系统风格</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="styleCombo">
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>定义风格</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QComboBox" name="styleSheetCombo">
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
